backend/lcfs/web/api/compliance_report/repo.py

```diff
@@ -667,18 +668,16 @@ async def get_issued_compliance_units(
 
     @repo_handler
     async def calculate_fuel_quantities(
```

Comment:
Just a suggestion, is it possible to separate the business logic into service component and use repo module only for data extraction?

Reply:
Broke this into multiple functions. The calculate is now in the service and it uses 3 query methods from the repo for the logic.